packages/datetime/test/dateInputTests.tsx

```diff
@@ -63,6 +63,31 @@ describe("<DateInput>", () => {
         assert.isFalse(wrapper.find(Popover).prop("isOpen"));
     });
 
+    it("Popover closes when ESC key pressed", () => {
```

Comment:
`Popover` provides this feature, and has corresponding tests. can we leverage it instead of reimplementing/testing here?
https://github.com/palantir/blueprint/blob/develop/packages/core/test/popover/popoverTests.tsx#L565

Reply:
- [ ] use popover's close implementation, remove this test

Reply:
As mentioned up top, we still need the functionality for hitting ESC while the input is in focus. So I still think we need this.